### PR TITLE
#346 Add mark as unread btn

### PR DESCRIPTION
**Pull request by oubchid**
https://user-images.githubusercontent.com/7097550/109018936-d4222680-76b8-11eb-940e-23c3c482b1a0.mp4

### CODE REVIEW

File: src/linagora.esn.unifiedinbox/views/unified-inbox/subheader.pug [hunk 1]
```diff
@@ -17,5 +17,6 @@ block right
     .inbox-selection-aware-subheader.inbox-selecting(ng-switch-when="true")
       inbox-subheader-not-spam-button(ng-click="ctrl.unSpam()", ng-show="ctrl.canUnSpamMessages()")
       inbox-subheader-delete-button(ng-click="ctrl.moveToTrash()", ng-show="ctrl.canTrashMessages()")
-      inbox-subheader-mark-as-read-button(ng-click="ctrl.markAsRead()")
+      inbox-subheader-mark-as-read-button(ng-click="ctrl.markAsRead()", ng-if="ctrl.isUnreadItems() || ctrl.mixedItems()")
+      inbox-subheader-mark-as-unread-button(ng-click="ctrl.markAsUnread()", ng-if="ctrl.isReadItems() || ctrl.mixedItems()")
```
Comment (alagane) [inline src/linagora.esn.unifiedinbox/views/unified-inbox/subheader.pug:21]: I find it easier to:
* show the markAsRead button if isAtLeastOneUnreadItem
* show the markAsUnread button if isAtLeastOneReadItem

Comment (oubchid) [reply src/linagora.esn.unifiedinbox/views/unified-inbox/subheader.pug:21]: easier what do you mean ?

Comment (alagane) [reply src/linagora.esn.unifiedinbox/views/unified-inbox/subheader.pug:21]: I mean easier to understand, and more optimized, simpler.
If you have to explain it you would say "The button to mark as read is here if at least on mail is unread" and "The button to mark as unread is here if at least on mail is read", and that makes sense in my opinion.
So yes it means removing the mixedItems and modifying and renaming the 2 other methods.

Comment (oubchid) [reply src/linagora.esn.unifiedinbox/views/unified-inbox/subheader.pug:21]: the first one is easy to understand
I will keep it more simpler

**Review by alagane — APPROVED** [review]

**Review by fabienmoyon — APPROVED** [review]

**Pull request merged**